spec/spec.md

```diff
@@ -142,7 +142,7 @@ sequenceDiagram
 - As the query language, Presentation Exchange MUST be used and conform to the syntax defined in [[ref: OpenID4VP]], a profile of [[ref: Presentation Exchange]].
 - Decentralized Identifiers (DIDs), as defined in [[ref: DID Core]], MUST be used as identifiers of the entities. Implementations MUST support did:web and did:ion as a mandatory DID method as defined in [[ref: did-web]] and [[ref: did-ion]].
 - To bind an owner of a DID to a controller of a certain origin, a Well Known DID Configuration MUST be used as defined in [[ref: Well Known DID]].
-- For Revocation of VCs, Status List 2021 as defined in [[ref: Status List 2021]] MUST be used in combination with Identity Hubs as defined in [[ref: Identity Hub]] (Decentralized Web Node v0.0.1 predraft).
+- For Revocation of VCs Status List 2021 as defined in [[ref: Status List 2021]] MAY be used. If used it MUST be resolvable from either a HTTPS web resource or an ID Hub as defined in [[ref: Identity Hub]] (Decentralized Web Node v0.0.1 predraft).
```

Comment:
Could we change the language to align with line 466:

```suggestion
- For Revocation of VCs, Status List 2021 as defined in [[ref: Status List 2021]] MAY be used. If used, it MUST be resolvable from either a DID Relative URLs stored in an ID Hub as defined in [[ref: Identity Hub]] (Decentralized Web Node v0.0.1 predraft), or a HTTPS URLs.
```

(I would also reverse the order because it can be read as if HTTPS URLs can be stored in ID Hubs)...

Reply:
and maybe change line 466 to 
`StatusList2021 MUST be discovered using either DID Relative URLs stored in an ID Hub or  HTTPS URL.`

Reply:
updated